### ISSUE TITLE
Some ui-config settings are not working

### ISSUE THREAD

**Issue by westNeighbor** [lllyasviel/stable-diffusion-webui-forge#1100]
I only tried to change several default values, but found the followings don't work
    "customscript/sampler.py/txt2img/Sampling method/value": "DPM++ 2M",
    "customscript/sampler.py/txt2img/Schedule type/value": "Karras",
    "txt2img/Height/value": 768,
    "txt2img/CFG Scale/value": 3.0,

**Comment by leukophobia** [comment]
Yes they are broken, same for me. Just did a clean install and update.

**Comment by juicyroots** [comment]
Same issue here, can't set a handful of options in ui-config.json, but others work. Sampling method, schedule type, and check-boxes seem to be the issues for me.

version: [f2.0.1v1.10.1-previous-320-gdb5a876d](https://github.com/lllyasviel/stable-diffusion-webui-forge/commit/db5a876d4cdadccf921c730d3167de25375a61c7)  •  python: 3.10.13  •  torch: 2.4.0+cu124  •  xformers: N/A  •  gradio: 4.40.0
